iree/tools/run_mlir_main.cc

```diff
@@ -267,11 +289,20 @@ Status EvaluateFunction(iree_vm_context_t* context,
 
   std::cout << "EXEC @" << export_name << std::endl;
   ASSIGN_OR_RETURN(auto input_descs, ParseInputSignature(function));
-  auto input_values_list = absl::MakeConstSpan(
-      input_values_flag.empty() ? nullptr : &input_values_flag.front(),
-      input_values_flag.size());
-  ASSIGN_OR_RETURN(auto* input_list, ParseToVariantList(input_descs, allocator,
-                                                        input_values_list));
+  iree_vm_variant_list_t* input_list;
+  if (!input_values_file_flag.empty()) {
+    ASSIGN_OR_RETURN(auto input_values, GetInputValues(input_values_file_flag));
+    ASSIGN_OR_RETURN(input_list, ParseToVariantList(
+                                     input_descs, allocator,
+                                     absl::MakeConstSpan(&input_values.front(),
+                                                         input_values.size())));
```

Comment:
Seems like explicit span creation here shouldn't be necessary. There should be an implicit conversion from vector to const span

Reply:
Oh I didn't find it, thanks!